app/Platform/Concerns/ActsAsDeveloper.php

```diff
@@ -26,6 +27,14 @@ public function authoredAchievements(): HasMany
         return $this->hasMany(Achievement::class, 'Author', 'User');
     }
 
+    /**
+     * @return HasMany<MemoryNote>
+     */
+    public function authoredCodeNotes(): HasMany
+    {
+        return $this->hasMany(MemoryNote::class, 'AuthorID', 'ID');
```

Comment:
I'd probably put the "where not empty" here, as empty notes represent deleted notes.

Reply:
Yes, agreed. Done in latest.